jit/v2/JIT_v2.0.0.yaml

```diff
@@ -2086,14 +2086,47 @@ components:
     #######
     # Moves
     #######
+    MovesList:
+      type: array
+      description: |
+        An array of `Moves`. Can be per VSA-Partner or as an accumulated result.
+      items:
+        $ref: '#/components/schemas/Moves'
     Moves:
       type: object
       title: Moves
       description: |
         Forecasted container moves for loading and discharging, empty or laden, at a terminal, covering various types such as Dry, Reefer, Out-of-Gauge (OOG), and Restows. These can be detailed at varying levels of granularity.
 
         **Condition:** Can only be provided if `portCallServiceType` is `MOVES`
+
+        **Condition:** `carrierCode` and `carrierCodeListProvider` are to be used if the moves provided in this object is **only** for a single VSA-Partner. If this object is provided **without** `carrierCode` and `carrierCodeListProvider` then it means it is an aggregated result for the remaining moves.
       properties:
+        carrierCode:
+          type: string
+          description: |
+            Providing this property means that the `Moves` is only related to this particular `carrierCode`. Omitting this property means that the `Moves` is an aggregated result from all VSA-Partners.
+
+            The carrier code based on either the **SMDG** or **SCAC** code lists.
+
+            **Condition:** This is required if `carrierCodeListProvider` is provided
+          maxLength: 4
+          pattern: ^\S+$
+          example: MAEU
+        carrierCodeListProvider:
+          type: string
+          description: |
+            Providing this property means that the `Moves` is only related to this particular `carrierCodeListProvider`. Omitting this property means that the `Moves` is an aggregated result from all VSA-Partners.
```

Comment:
For the carrier code list provider. I don't think we need the line of: "Providing this property means that..."

This property will only be provided if the carrier code is provided and the carrier code already explains what it means "Providing this property..."